.github/workflows/validate.yml

```diff
@@ -16,19 +16,17 @@ jobs:
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.18, 10, 12, 14, 15]
+        node: [10.18, 10, 12, 14, 16]
```

Comment:
Node 15 is still supported until June 1
```suggestion
        node: [10.18, 10, 12, 14, 15, 16]
```

Reply:
We shouldn't support odd Node versions, only when they're `latest`

Reply:
As far as I understand from [Node's LTS schedule](https://github.com/nodejs/Release), 15 is actually still supported right now, even though 16 (which will eventually be upgraded to an LTS) has been released since. Technically 16 is the only latest release here anyway.

Reply:
15 is officially EOL now.